splice_access_api/src/main/java/com/splicemachine/access/configuration/SConfigurationImpl.java

```diff
@@ -276,7 +277,10 @@ public boolean getAuthenticationTokenDebugConnections() {
     public int getAuthenticationTokenMaxConnections() {
         return authenticationTokenMaxConnections;
     }
-
+    @Override
+    public int getAuthenticationTokenPermissionCacheSize() {
```

Comment:
This configuration parameter does not seem to be used anywhere. Did I miss anything?

Reply:
@yxia92 It's used on the -ee side, I forgot to create the PR, sorry!